packages/flutter_tools/lib/src/web/devfs_web.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:meta/meta.dart';
+import 'package:mime/mime.dart' as mime;
+
+import '../base/common.dart';
+import '../base/file_system.dart';
+import '../base/io.dart';
+import '../build_info.dart';
+import '../convert.dart';
+import '../globals.dart';
+
+/// A web server which handles serving JavaScript and assets.
+///
+/// This is only used in development mode.
+class WebAssetServer {
+  @visibleForTesting
+  WebAssetServer(this._httpServer) {
+    _httpServer.listen((HttpRequest request) {
+      try {
+        return _handleRequest(request);
```

Comment:
Listen callbacks are expected to return void.

Reply:
Fixed